clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ArtifactController.java

```diff
@@ -58,26 +57,12 @@ public ArtifactController(Optional<ArtifactCredentialsRepository> artifactCreden
     }
   }
 
-  @RequestMapping(method = RequestMethod.GET, value = "/fetch")
-  String fetch(@RequestParam("artifactAccount") String artifactAccount,
-      @RequestParam("type") String type,
-      @RequestParam("reference") String reference,
-      @RequestParam(value = "version", required = false) String version) {
+  @RequestMapping(method = RequestMethod.PUT, value = "/fetch")
```

Comment:
Can you comment why this is a `PUT` (like in your commit history)? Looks strange without the context in this PR.